src/shared/global.ts

```diff
@@ -50,7 +54,8 @@ export const GlobalStyles = createGlobalStyle`
     --spacing-64: 64px;
 
     /* Borders */
-    --border: 1px solid #9E9E9E;
+    --border-primary: 1px solid #9E9E9E;
+    --border-error: 2px solid var(--color-alertRed);
```

Comment:
Can you get problems toggeling between these borders when one is 2px and the other 1px? I think you can get jumping because one is bigger?

Reply:
Yeah, should be inside. Will check if I set the correct box-sizing